ops/model.py

```diff
@@ -2509,7 +2509,15 @@ def remove_path(self, path: Union[str, PurePath], *, recursive: bool = False):
 
         Args:
             path: Path of the file or directory to delete from the remote system.
-            recursive: If True, recursively delete path and everything under it.
+            recursive: If True, and path is a directory, recursively deletes it and
+                       everything under it. If the path is a file, delete the file and
+                       do nothing if the file is non-existent. Behaviourally similar
+                       to ``rm -rf <file|dir>``.
+
+        Raises:
+            pebble.PathError: If the file cannot be removed (perhaps it does not
+                exist) and `recursive` is false, or if a relative path is provided.
```

Comment:
This wording is also awkward. Perhaps:
```suggestion
            pebble.PathError: If a relative path is provided, or if `recursive` is False
                and the file or directory cannot be removed (it does not exist or is not empty).
```